### DISCUSSION TITLE
Quiet on updates (Health issues)

### DISCUSSION THREAD

**Discussion by magicbug** [magicbug/Cloudlog#1000] [General]
Hi All,

Just to say that things might be a little slow regarding Cloudlog development for the next couple of weeks, I've been in and out of hospital but back home and recovering but slowly but will be some time before I'm back at my peak.

Cloudlog isn't going anywhere, just need to focus a bit on health :)

Peter.

**Comment by tonyc** [comment]
Cheers, Pete, your health is definitely the most important thing!

**Comment by gwince** [comment]
Take care of you first, Pete. :)

**Comment by gmcintire** [comment]
Take whatever time you need to get better, your health is worth it!

**Comment by AndreasK79** [comment]
Glad you are recovering Peter. I've been trying to assist here with whatever I can contribute with.

**Reply by magicbug** [reply]
@AndreasK79 The help you have given is really appreciated just so you know.. healths an up hill struggle but things seem to be turning for the better.
